README.rst

```diff
@@ -14,19 +14,23 @@ Project DeepSpeech
 
 DeepSpeech is an open source Speech-To-Text engine, using a model trained by machine learning techniques based on `Baidu's Deep Speech research paper <https://arxiv.org/abs/1412.5567>`_. Project DeepSpeech uses Google's `TensorFlow <https://www.tensorflow.org/>`_ to make the implementation easier.
 
-**NOTE:** This documentation applies to the **MASTER version** of DeepSpeech only. **Documentation for the latest stable version** is published on `deepspeech.readthedocs.io <http://deepspeech.readthedocs.io/?badge=latest>`_.
+To install and use the **latest stable version**, please see the `DeepSpeech install guide <http://deepspeech.readthedocs.io/?badge=latest>`_  available at `deepspeech.readthedocs.io <http://deepspeech.readthedocs.io/?badge=latest>`_.
 
-To install and use deepspeech all you have to do is:
+To install the current stable release:
+
+.. code-block:: bash
+
+   # Install DeepSpeech
+   pip3 install deepspeech
+
+If you're using the **MASTER version** of DeepSpeech, you can get started using a pre-trained model:
```

Comment:
This seems to imply that there exists a pre-trained model for the MASTER version which is not the case.

Reply:
Yeah, I wasn't sure what to do with this part. We could either:
1. delete these instructions
2. reword to "If you're using the **MASTER version** of DeepSpeech, here is the syntax for using a pre-trained model:"

Reply:
To me this seems more confusing than it was at the start.

Now this documentation has to juggle two versions of the code. Where previously it had a big warning that said go to readthedocs for the latest stable version, otherwise stay here for Master.

Reply:
Thanks Kelly - I'm trying to wrap my head around why this block of code exists? It mixes installing the stable version, using 0.6.1 models, and master syntax. When will someone be able to run this code?

I'm trying to separate it out into chunks that are usable, but that seems to be creating its own problems.

Reply:
>  When will someone be able to run this code?

This is accurate at release time.

Reply:
Your GitHub landing page (and hence master README) is likely the first thing someone will see about this project. Highlighting readthedocs is great, but there will definitely be people skipping to the first block of code to get started.

Could this be something that gets added at release time? Rather than having potentially inaccurate code being the first thing people see?

Reply:
The code as currently written will never work, even after 0.7 is released, because it mixes 0.7 syntax with the 0.6 models.

Isn’t it easier to just quickly change the docs right before a release than have non-working code be the first thing users see?

Reply:
I really really don't understand what is so complicated for people to change a tag / version in Github. This is not targetting random John Doe, this is targetting developpers. Even students should have the basics knowledge and reflex to change versions if we mention that.

> Isn’t it easier to just quickly change the docs right before a release than have non-working code be the first thing users see?

Yes, it's another-extra-manual-step that makes is error prone and painful.

Reply:
We already have a manual step to update all the model references for the new version.

Reply:
Thanks @lissyx -- I'm just trying to lower the barrier of entry and make this project as accessible as possible. I have a CS degree and over a decade of experience and this still tripped me up for a few minutes.

I don't want to step on any toes, if you think this project is accessible as-is, I'm happy to step away.

Reply:
> Thanks @lissyx -- I'm just trying to lower the barrier of entry and make this project as accessible as possible. I have a CS degree and over a decade of experience and this still tripped me up for a few minutes.
> 
> I don't want to step on any toes, if you think this project is accessible as-is, I'm happy to step away.

That's why it's important that we understand why what seems to be so obvious to us and many other people is not to some others.

Reply:
@acabunoc Did you tripped over this because of the amount of docs, or because it's unclear, or because of something else ?

Reply:
Thanks for asking @lissyx --

I made the assumption that the first block of code on your landing page would be a good place to get started. I understand that there are instructions to look elsewhere, and I eventually saw that and moved to the docs. However, I will not be the only person making this assumption. Most thriving open source projects will have a usable block of code on their landing page, if they have code there. I'd love to see more developers stepping into this project and immediately feeling comfortable.

Tiny changes like this can make a huge different in community engagement and go long way to show new contributors that their presence is welcome here.

Reply:
>  Most thriving open source projects will have a usable block of code on their landing page, if they have code there. I'd love to see more developers stepping into this project and immediately feeling comfortable.

Well, I share that. At the same time, everytime I'm looking at some docs from a `master` branch, I know it might fail, and if it does, I look for something that is stable.

I think we all on the team assumed this was the default behavior of people. Is this wrong ? Do people don't care about matching-version documentations ?

I don't see how we can ensure and provide always-working master branch + instructions, especially given a release is made when a new model is ready.


> I made the assumption that the first block of code on your landing page would be a good place to get started. I understand that there are instructions to look elsewhere, and I eventually saw that and moved to the docs.

The obvious solution would be to keep "stable" instructions on top, but then it is also confusing to people in other ways. And it's also going to be much more messy to mix.

I honestly have no idea how we can fix everything at once.

Reply:
We don't provide always working master branch instructions. As it is, our master branch is wrong most of the time. It's only correct in the time between a stable release and the first breaking change. As far as I can see, the only benefit our policy brings is to reduce the number of changes we have to do at release time (just update the model URL). Personally I'm fine with leaving the master instructions matching the latest stable release.

Another possibility is to create release branches in addition to release tags, and then switching the GitHub default branch after a release. That way when users come to the repo the stable instructions are shown, and we make it so that only contributors need to do the extra work of switching versions. I guess people who know enough to change the code and send PRs also know enough that changing versions won't be a hassle.

Reply:
Thanks both!

> Well, I share that. At the same time, everytime I'm looking at some docs from a master branch, I know it might fail, and if it does, I look for something that is stable.
> 
> I think we all on the team assumed this was the default behavior of people. Is this wrong ? Do people don't care about matching-version documentations ?

In many cases, I think you're right. But this is also the main landing page for your project -- not just some docs. Small barriers here can signal that that you have your own short-hand on how you work as a team & you're not ready for outside contributors.

I agree that people experienced in open source & code can easily get over this. But I would hope that this project could be welcoming to more than the existing open source crowd (which has its own diversity problems).

@reuben - huge thanks for bringing up a couple solutions. Both are great. The team knows more about your workflow/community to know what would work best here. Let me know if I can help at all.